modules/seastar/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+    "homepage": "https://github.com/scylladb/seastar",
+    "maintainers": [
+        {
+            "email": "noah@redpanda.com",
+            "github": "dotnwat",
+            "name": "Noah Watkins",
+            "github_user_id": 242417
+        }
+    ],
+    "repository": [
+        "github:scylladb/seastar"
+    ],
+    "versions": [
+        "25.08.0"
```

Comment:
Looks like this is a made up version that doesn't match any upstream tags? Can we make up a pseudo version according to https://github.com/bazelbuild/bazel-central-registry/blob/main/docs/README.md#pseudo-versions?

Reply:
Thanks, I've switched over to using a pseudo-version. Upstream does tag occasionally, but the latest is rather old. There hasn't been a lot of demand for that (most people use upstream@HEAD) but Bazel may drive more consistent table tags and we'll be working on that.